CapMan.Raylib/BoardRenderer.cs

```diff
@@ -34,14 +34,14 @@ private static string AssetPath(Element element)
     {
         return "assets/sprites/" + element switch
         {
-            Element.Horizontal => "horizontal.png",
-            Element.Vertical => "vertical.png",
-            Element.TopLeft => "top-left.png",
-            Element.TopRight => "top-right.png",
-            Element.BottomLeft => "bottom-left.png",
+            Element.Horizontal  => "horizontal.png",
+            Element.Vertical    => "vertical.png",
+            Element.TopLeft     => "top-left.png",
+            Element.TopRight    => "top-right.png",
+            Element.BottomLeft  => "bottom-left.png",
             Element.BottomRight => "bottom-right.png",
-            Element.Dot => "dot.png",
-            Element.PowerPill => "powerpill.png",
+            Element.Dot         => "dot.png",
+            Element.PowerPill   => "powerpill.png",
```

Comment:
Does your formatter automatically add in this space?

Reply:
No, I just think it looks pretty. I can remove it.